test/scripts/plot_config.yaml

```diff
@@ -0,0 +1,4 @@
+data_filename: "../data/go1.csv"  # Dataset containing GRF measurements
+
+### CONVETION: start from right to left. from front to back FR FL RR RL. Fill the list in that order 
```

Comment:
why do we need a convention?

Reply:
Because the data are plotted with order FR FL RR RL. and this is specified by force_column_indxs. So this is practically a placeholder for the plot_data.py to return the force bias for each leg instead of only one bias from every one. Currently the order does not matter as long as you provide the columns that contain the forces.

Reply:
I suppose the comment  can be removed for now since the convention is not currently being used